rwgame/GameConfig.cpp

```diff
@@ -155,6 +155,7 @@ bool GameConfig::parseConfig(
                     } else {
                       success = false;
                       RW_MESSAGE(e.what());
+                      return;
                     }
                 }
```

Comment:
I included your suggestions in the next patch.
It also fixes a crash when parsing non-sensical data.